src/components/AudioTrack/layouts/Row.vue

```diff
@@ -7,7 +7,7 @@
       class="flex"
       :class="isSmall ? 'flex-row gap-8' : 'flex-col justify-between'"
     >
-      <div class="flex-shrink-0">
+      <div class="flex-shrink-0" :class="isSmall ? 'w-96' : ''">
```

Comment:
Instead of a fixed width, maybe a `max-width` or `min-width` would be preferable? Using hardcoded widths might negatively affect responsiveness.

Reply:
I was thinking the same but sadly with `min-w-` or `max-` the [tracks are misaligned](https://cln.sh/mTkrfg).